compiler/src-non-bootstrapped/scala/quoted/runtime/impl/QuotesImpl.scala

```diff
@@ -261,14 +261,14 @@ class QuotesImpl private (using val ctx: Context) extends Quotes, QuoteUnpickler
         }))
       def copy(original: Tree)(name: String, typeParams: List[TypeDef], paramss: List[List[ValDef]], tpt: TypeTree, rhs: Option[Term]): DefDef =
         tpd.cpy.DefDef(original)(name.toTermName, tpd.joinParams(typeParams, paramss), tpt, yCheckedOwners(rhs, original.symbol).getOrElse(tpd.EmptyTree))
-      def unapply(ddef: DefDef): (String, List[TypeDef], List[List[ValDef]], TypeTree, Option[Term]) =
-        (ddef.name.toString, ddef.typeParams, ddef.termParamss, ddef.tpt, optional(ddef.rhs))
+      // def unapply(ddef: DefDef): (String, List[TypeDef], List[List[ValDef]], TypeTree, Option[Term]) =
+      //   (ddef.name.toString, ddef.typeParams, ddef.termParamss, ddef.tpt, optional(ddef.rhs))
```

Comment:
Are the changes intentional?

Reply:
Yes, it is in the `src-non-bootstrapped` so the code does not matter to much as long as the compiler sees the required interfaces. I could have done it in a cleaner way but it would have required moving the `Matcher` to `src-bootstrapped`/`src-non-bootstrapped`. This source will be removed soon when we re-bootstrap the compiler.